packages/crml-attestation/package.json

```diff
@@ -13,12 +13,12 @@
     },
     "scripts": {},
     "dependencies": {
-        "@cennznet/api": "^1.0.0",
-        "@cennznet/types": "^1.0.0",
-        "@cennznet/util": "^1.0.0"
+        "@cennznet/api": "1.0.0",
```

Comment:
Does it mean every time we release a new API, we should remember to manually update this line too?